apps/examples/mdns_test/mdns_main.c

```diff
@@ -68,8 +70,13 @@
 /****************************************************************************
  * Definitions
  ****************************************************************************/
-#define MDNS_HOST_NAME          "tinyara.local"
+#if defined(CONFIG_ARCH_BOARD_SIDK_S5JT200) || defined(CONFIG_ARCH_BOARD_ARTIK053)
```

Comment:
We will likely have many variants that are based on S5JT200. So, lines 73 should be:
#if defined(CONFIG_ARCH_CHIP_S5JT200)